src/storage/MMKVStorage.ts

```diff
@@ -4,10 +4,10 @@ import { initializeMMKVFlipper } from 'react-native-mmkv-flipper-plugin'
 export type AcceptedValue = boolean | string | number | object
 
 export class MMKVStorage {
-  private id = 'mmkv.default'
-  private storage: MMKV | null = null
+  private id: string
+  private storage: MMKV
 
-  constructor(id = 'mmkv.default', encryptionKey?: string) {
+  constructor(id = 'main_storage', encryptionKey?: string) {
```

Comment:
This should not change, else you'll nuke the entire mmkv storage as you're pointing it to a new key.